src/charts/brush-range-editor.js

```diff
@@ -65,10 +87,32 @@ export default function BrushRangeEditor (_container) {
       cache.inputMin = cache.root.append("div")
         .attr("class", "brush-range-input min")
         .attr("contentEditable", true)
+        .on("focus", function focus () {
+          let text = cache.inputMin.text()
+          const parsed = d3.timeParse("%b %d, %Y")(text)
```

Comment:
I guess you could use `config.dateFormat` here instead of a hardcoded format, to match the `cache.inputMin.text(d3.utcFormat(config.dateFormat)(oldValue))` below